apps/examples/mediarecorder/mediarecorder_main.cpp

```diff
@@ -275,6 +308,7 @@ class MediaRecorderTest : public MediaRecorderObserverInterface, public enable_s
 	struct pcm *p_out;
```

Comment:
Remove the settings and member variables associated with the pcmConfig structure.

Reply:
Fixed